gulpfile.js

```diff
@@ -104,7 +104,7 @@ var cssTasks = function(filename) {
         'opera 12'
       ]
     })
-    .pipe(minifyCss, {
+    .pipe(cssNano, {
       advanced: false,
```

Comment:
While you are revising the options: basically disable all "clever" options. Potentially destructive optimizations are not wanted.